manifests/service.pp

```diff
@@ -41,8 +41,8 @@
 ) {
   $nexus_script = "${nexus_home}/bin/nexus"
 
-  if $::operatingsystem == 'Debian' {
-    if versioncmp($::operatingsystemrelease, '8.0') > 0 {
+  if $::operatingsystem == 'Debian' or $::operatingsystem == 'Ubuntu' {
+    if versioncmp($::operatingsystemrelease, '8.0') > 0 and $::operatingsystem == 'Debian' {
```

Comment:
I don't think this change does what you want, the inner if-statement is:
```
if versioncmp($::operatingsystemrelease, '8.0') > 0 and $::operatingsystem == 'Debian' {
```
which will never evaluate to `true` unless `$::operatingsystem == 'Debian'`.

I think these two if-statements should be merged into a single if-statement:
```
($::operatingsystem == 'Debian' and versioncmp($::operatingsystemrelease, '8.0') > 0) or $::operatingsystem == 'Ubuntu'
```